Client/Coordinators/Browser/BrowserCoordinator.swift

```diff
@@ -267,7 +267,9 @@ class BrowserCoordinator: BaseCoordinator, LaunchCoordinatorDelegate, BrowserDel
         }
 
         let navigationController = ThemedNavigationController()
-        navigationController.modalPresentationStyle = .formSheet
+        let isPad = UIDevice.current.userInterfaceIdiom == .pad
```

Comment:
Should we maybe check the size class instead of the device here? With multitasking & stage manager we could end up displaying the iPhone UI on iPad as well.

Reply:
We discussed, and determined for this case this is what we'll want